ONSdigital/census-rm-ops-ui
# 1221 Search for case and show results

# Motivation and Context

The first feature needed for ops-ui is to be able to search for cases via a postcode. This PR adds that functionality as well as IAP Audit logging and adding the CDN to ops-ui.

# What has changed



- Search for cases via postcode
- IAP audit logging
- Added flask/jinja templates to show postcode results.
- Added Flask blueprints for routing
- Now runs with Gunicorn

# How to test?



- Run with the other branches on the ticket. When you have cases in your database, you should be able to run the ops-ui and find cases with a postcode.
- Try running this in your GCP environment and see if you can connect to it.
- Anyone testing I can add to the IAP so you could try and connect to the domain set up in `census-rm-ryangrundy`

# Links



[Trello](https://trello.com/c/qkkjvupd)
